app.rb

```diff
@@ -0,0 +1,159 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  attr_accessor :books, :persons, :rentals
+
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def begin
+    puts "Welcome to School Library App!\n"
+    command = 0
+    while command != 7
+      show_menu
+      command = gets.chomp
+      operate(command)
+    end
+  end
+
+  def show_menu
+    puts 'Please choose an option by entring a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def operate(command)
+    case command
+    when '1'
+      list_books
```

Comment:
- [ ] While developing software, the separation of concern is a very crucial point for it allows readability of your codebase and make it more organized. as specified in the project requirements kindly make sure the methods `operate`, `showmenu` and `begin` are inside the `main` method and the method is in a file called `main.rb`